### DISCUSSION TITLE
DOH on UDM

### DISCUSSION THREAD

**Discussion by zs311521** [TechnitiumSoftware/DnsServer#1047] [Q&A]
Hi, Unifi has a DOH feature.

I was thinking, since I Have my own DNS resolver, to forward directly through UDM rather than having a local Technetium instance, this should maintain speed even though queries will not be cached locally as one less hop.

But, how do I set up technetium - will this work? DNS stamp, what would I put into the server name? Thank you so much.

<img width="817" alt="image" src="https://github.com/user-attachments/assets/44c8210c-2e37-4247-8f28-b047fee186ec">

**Comment by ShreyasZare** [comment]
Thanks for asking. There is no support for DNS Stamp. Will see how something like this can be supported.

**Comment by IngmarStein** [comment]
Can't you generate a stamp using https://dnscrypt.info/stamps/ and put whatever you want as the server name?

**Reply by Avsynthe** [reply]
> Can't you generate a stamp using https://dnscrypt.info/stamps/ and put whatever you want as the server name?

Did you ever figure this out? I'm using traefik to handle TLS/SSL termination the DoH and DoT ports on Technitium with doh.mydomain.com and dot.mydomain.com respectively.

I can't quite figure out how to configure the stamp at https://dnscrypt.info/stamps/. I'm sure I had it right but I don't believe its working
